apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -779,6 +779,7 @@ SOMAExperimentAxisQuery <- R6::R6Class(
           i = self$indexer$by_obs(obs)$as_vector() + 1L,
```

Comment:
Looking at this again, is this internal `as_matrix()` function redundant with `private$as_matrix()`?

Reply:
Yes? I had built `private$.as_matrix()` to try to share code between the assay, dimensional reduction, and graph methods. However, DRs are dense and graphs have a shape of `[obs, obs]` instead of `[obs, var]` so sharing was moot. In the future, `private$.as_matrix()` might be useful for converting to v5 assays but as of now it should be removed in favor of the internal `as_matrix()`

(I forgot about it in the original PR and didn't want to expand the scope of this one, but can if you want)